### ISSUE TITLE
Fix EE deploy template to use nodejs 14 to prevent consistently failing

### ISSUE THREAD

**Issue by rudpot** [aws-samples/aws-fault-injection-simulator-workshop#333]
Root cause TBD. Manually running deploy-parallel.sh from EE account works but running deploy using the codebuild wrapper fails to instantiate VPC.

**Comment by rudpot** [comment]
```


--- deploy-output.vpc.txt ---
--
615 | Provisioning vpc resources
616 | npm WARN read-shrinkwrap This version of npm is compatible with lockfileVersion@1, but package-lock.json was generated for lockfileVersion@2. I'll try to do my best with it!
617 | npm WARN vpc@0.1.0 No repository field.
618 | npm WARN vpc@0.1.0 No license field.
619 | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules/fsevents):
620 | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
621 |  
622 | added 355 packages from 268 contributors and audited 356 packages in 14.266s
623 |  
624 | 28 packages are looking for funding
625 | run `npm fund` for details
626 |  
627 | found 0 vulnerabilities
628 |  
629 | !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
630 | !!                                                                            !!
631 | !!  Node 12 has reached end-of-life on 2022-04-30 and is not supported.       !!
632 | !!  Please upgrade to a supported node version as soon as possible.           !!
633 | !!                                                                            !!
634 | !!  This software is currently running on node v12.22.12.                     !!
635 | !!  As of the current release of this software, supported node releases are:  !!
636 | !!  - ^18.0.0 (Planned end-of-life: 2025-04-30)                               !!
637 | !!  - ^16.3.0 (Planned end-of-life: 2024-04-30)                               !!
638 | !!  - ^14.5.0 (Planned end-of-life: 2023-04-30)                               !!
639 | !!                                                                            !!
640 | !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
641 | Unexpected token '?'
642 |  
643 | Subprocess exited with error 1

```

**Comment by rudpot** [comment]
Removed offending commit #332 but not clear why this caused an issue

**Comment by rudpot** [comment]
Issue was caused by CDK no longer supporting nodejs 12. Github actions test container runs updated version of nodejs but deployment script doesn't.

**Comment by rudpot** [comment]
This can probably be solved by changing both the GitHub Actions and CodeBuild containers to use Ubuntu.

**Comment by rudpot** [comment]
Fixed by #336
